.travis.yml

```diff
@@ -31,12 +31,22 @@ jobs:
       sudo: required
       env: TOXENV=py37
 
+    -
+      python: 3.8
+      dist: xenial
+      env: TOXENV=py38
+
     -
       python: 3.7
       dist: xenial
       sudo: required
       env: TOXENV=py37-ansi2html
 
+    -
+      python: 3.8
+      dist: xenial
```

Comment:
`dist: xenial` is the default, so could be removed from all of these.

Also `sudo:` no longer does anything and can be removed: https://blog.travis-ci.com/2018-11-19-required-linux-infrastructure-migration